python_modules/dagit/dagit/schema.py

```diff
@@ -404,6 +427,9 @@ def __init__(self, dagster_type):
         )
         self._dagster_type = dagster_type
 
+    def resolve_type_attributes(self, _info):
+        return self._dagster_type.type_attributes
```

Comment:
I actually didn't realize Graphene would accept something matching the expected interface that wasn't actually an instance of the `TypeAttributes` class, that's great.